kiwi/tasks/base.py

```diff
@@ -177,6 +178,21 @@ def sextuple_token(self, option):
             )
         ]
 
+    def run_checks(self, checks):
+        """
+        This method runs the given runtime checks excluding the ones disabled
+        in the runtime configuration file.
+
+        :param dict checks: A dictionary with the runtime method names as keys
+            and their arguments list as the values.
+        """
+        exclude_list = self.runtime_config.get_disabled_runtime_checks()
+        if self.runtime_checker is not None:
+            for method, args in {
+                k: v for k, v in checks.items() if k not in exclude_list
```

Comment:
nitpick, can we avoid variables names whose scope is only obvious after reading the code ? I mean k is key and v is value, which would be a nicer read imho

Reply:
sure let me try with better names